js/initcontext.go

```diff
@@ -385,6 +386,7 @@ func getInternalJSModules() map[string]interface{} {
 		"k6/html":        html.New(),
 		"k6/http":        http.New(),
 		"k6/metrics":     metrics.New(),
+		"k6/redis":       redis.New(),
```

Comment:
@oleiade I guess this is unexpected

Reply:
Woooooops. Good catch cheers 🙇🏻  (consider this a teaser).